learn/guides/distinctive-language-features/concurrency.md

```diff
@@ -433,7 +433,7 @@ In the above code example, the function **``update()``** has a transactional con
 
 This is particularly useful when the **``update()``** function is called from a remote or resource transactional method, in a service object, and is invoked by another remote Ballerina program as a result of a service invocation. The transaction manager of the Ballerina program that initiated the transaction, will send a message to the Ballerina program in which this function is running. The two transaction managers follow a two-phase commit, such that when the remote Ballerina program knows that commit is successful, it will then arrange for the transaction manager of the Ballerina program running the **``update()``** function to call the commit handler.
 
-## Concurency Safety
+## Concurrency Safety
```

Comment:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/3736/files#diff-3703f09cba3dd284401f066731bbec18a5d6cd760e6a609203e0e7dd50922306R436.

Reply:
Should be fine to have the commit here as well.